fetch.bs

```diff
@@ -3909,6 +3909,30 @@ these steps:
    <li><p>Let <var>requestForServiceWorker</var> be a <a for=request>clone</a> of
    <var>request</var>.
 
+   <li>
+    <p> If <var>requestForServiceWorker</var>'s <a for=/>body</a> is non-null, then:
+
+    <ol>
+     <li> Let <var>transformAlgorithm</var> given <var>chunk</var> be these steps:
+      <ol>
+        <li><p>If the ongoing fetch is <a for=fetch>terminated</a>, then abort these steps.
+
+        <li><p>If <var>chunk</var> is not a {{Uint8Array}} object, then
+         <a lt=terminated for=fetch>terminate</a> the ongoing fetch.
+
+        <li>Otherwise, <a for=ReadableStream>enqueue</a> <var>chunk</var>. The user agent may split
+         the chunk and enqueues multiple times.
+      </ol>
+
+     <li> Let <var>transformStream</var> be result of <a for=TransformStream>creating</a> a
+      {{TransformStream}} with <var>transformAlgorithm</var>.
+
+     <li> Let <var>streamForServiceWorker</var> be result of pipeThrough
```

Comment:
I don't know how to link https://streams.spec.whatwg.org/#rs-pipe-through

Reply:
We cannot use that as that's the public API. @ricea might be able to help here.

Reply:
You can use https://streams.spec.whatwg.org/#rs-pipe-through directly. Something like

1. Perform ReadableStreamPipeTo(the stream, _transformStream_.[[writable]]).
1. Let _streamForServiceWorker_ be _transformStream_.[[readable]].

Reply:
We could wrap this up into a helper algorithm similar to the others in https://streams.spec.whatwg.org/#other-specs so you can avoid violating the "shoulds" there. I will try to do so.

Reply:
Done.